libshaderc_spvc/include/shaderc/spvc.h

```diff
@@ -95,6 +95,10 @@ SHADERC_EXPORT void shaderc_spvc_compile_options_set_separate_shader_objects(
 SHADERC_EXPORT void shaderc_spvc_compile_options_set_flatten_ubo(
     shaderc_spvc_compile_options_t options, bool b);
 
+// Transform the input to WebGPU compatible SPIRV before cross-compiling.
+SHADERC_EXPORT void shaderc_spvc_compile_options_set_transform_to_webgpu(
```

Comment:
But... I don't expect SPIRV-Cross to consume a shader in WebGPU SPIR-V form.  I expected this MR to transform in the other direction: from WebGPU SPIR-V to Vulkan SPIR-V.... ?